src/main/scala/mesosphere/marathon/core/pod/Network.scala

```diff
@@ -0,0 +1,37 @@
+package mesosphere.marathon.core.pod
+
+import mesosphere.marathon.raml.{ KVLabels, NetworkMode, Network => RAMLNetwork }
+
+/**
+  * Network declared by a [[PodDefinition]].
+  */
+sealed trait Network extends Product with Serializable {
+  def toAPIObject: RAMLNetwork
```

Comment:
We start adding api level code into the core.
Wdyt putting this logic into a conversion class/package under api?